ports/nrf/modules/machine/pin.c

```diff
@@ -543,7 +551,7 @@ STATIC const mp_rom_map_elem_t pin_locals_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR_mode),    MP_ROM_PTR(&pin_mode_obj) },
     { MP_ROM_QSTR(MP_QSTR_pull),    MP_ROM_PTR(&pin_pull_obj) },
     { MP_ROM_QSTR(MP_QSTR_af),      MP_ROM_PTR(&pin_af_obj) },
-    { MP_ROM_QSTR(MP_QSTR_irq),     MP_ROM_PTR(&pin_irq_obj) },
+//    { MP_ROM_QSTR(MP_QSTR_irq),     MP_ROM_PTR(&pin_irq_obj) },
```

Comment:
Are pin interrupts not supported or simply not yet implemented?

Reply:
It's somewhat implemented. However, what is of implementation, commented out, works. Pin interrupt should be changed to port events before activated, so that all pins can be used and not limited by number of GPIOTE channels.